### PR TITLE
Upgrade to Statie 5.3

### PR DESCRIPTION
**Pull request by TomasVotruba**
New Statie 5 ships with human upgrade :)

Let me know if you preffer Twig over Latte.

### CODE REVIEW

File: source/_layouts/default.latte [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <html lang="cs">
-{include "head"}
+{include "_snippets/head.latte"}
```
Comment (TomasVotruba) [inline source/_layouts/default.latte:3]: Files need full relative paths, former "head" naming caused conflicts in files with same base name

File: source/index.latte [hunk 2]
```diff
@@ -288,4 +288,3 @@ title: "Tomáš Fejfar, PHP vývojář a lektor webového vývoje"
         </ul>
     </div>
 </div>
-{/block}
```
Comment (TomasVotruba) [inline source/index.latte:291]: Block is the whole file content, so no need to name it here

**Review by tomasfejfar — APPROVED** [review]
LGTM, thx

**Comment by tomasfejfar** [comment]
I am more comfortable with twig, but I don't mind latte. Or rather I don't think it's worth the hassle to change it. Anyway, thanks for the upgrade ;)

**Comment by TomasVotruba** [comment]
> Or rather I don't think it's worth the hassle to change it.

Not manually, [by the tool](https://github.com/symplify/lattetotwigconverter#latte-to-twig-converter) :) 

> Anyway, thanks for the upgrade ;)

:+1:

**Pull request merged**